Dockerfile

```diff
@@ -5,8 +5,9 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y wget cmake clang git autoconf pkg-config
 
 # Change submodule to https as we're cloning only. Avoids issues with ssh
+# 8bb209c0c21476ee904a is 0.4 with some bugfixes
 RUN mkdir /home/pdfalto && cd /home/pdfalto \
-    && git clone https://github.com/kermitt2/pdfalto.git && cd pdfalto && ./install_deps.sh \
+    && git clone https://github.com/kermitt2/pdfalto.git && cd pdfalto && git checkout 8bb209c0c21476ee904a && ./install_deps.sh \
```

Comment:
Yeah, it's _slooow_. I wasn't sure whether to fork, ended up opting not to as it would require maintaining a separate build but could be worth it in the long term for the reasons mentioned. I'll open an issue to capture.

Reply:
https://github.com/dlcs/pdf-to-alto/issues/2